src/utilities/new-sequence-editor/time-utils.ts

```diff
@@ -1,137 +1,212 @@
-export const ABSOLUTE_TIME = /^(\d{4})-(\d{3})T(\d{2}):(\d{2}):(\d{2})(?:\.(\d{3}))?$/g;
+import { CustomErrorCodes, type ErrorCode } from '../../workers/customCodes';
 
-export const RELATIVE_TIME = /([0-9]{3}T)?([0-9]{2}):([0-9]{2}):([0-9]{2})(\.[0-9]+)?$/g;
-export const RELATIVE_SIMPLE = /(\d+)(\.[0-9]+)?$/g;
+export enum TimeTypes {
```

Comment:
there is a `time.ts` enum file where this could probably be placed for easier discoverability